control-plane/tests/test_cli_inspection.py

```
@@ -281,6 +283,128 @@ def test_serve_command_uses_long_running_runtime_surface(self) -> None:
         build_runtime_service.assert_called_once_with()
         run_control_plane_service.assert_called_once_with(service, stderr=mock.ANY)
 
+    def test_startup_and_shutdown_status_commands_render_reviewed_contracts(self) -> None:
+        store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)